### PR TITLE
Add change user password and fix theme issue

### PR DESCRIPTION
**Pull request by ygbull**

### CODE REVIEW

File: src/style/AppTheme.js
Comment (Kolyn090) [inline src/style/AppTheme.js:1]: Solution to "theme.txt" not found. Write once before the first read.

**Review by Kolyn090 — APPROVED** [review]
Tested by 3 members.

File: src/controllers/profile_page/ResetPassword.js [hunk 1]
```diff
@@ -34,10 +39,57 @@ export function ProfilePageResetPasswordController() {
...
-  const setNewPassword = newPassword => {};
+        if (response.ok) {
+          console.log('Change sucesss');
+          onChangePageVariant(PageVariant.ChangeSuccess);
+        } else {
+          console.log('WrongOldPassword');
+          console.log(serverResponse);
+          console.log(userEmail);
+          console.log(passwordText);
+          console.log(repasswordText);
```
Comment (algebra2boy) [inline src/controllers/profile_page/ResetPassword.js:82]: should remove these logs

**Review by algebra2boy — COMMENTED** [review]
Looks good. Keep in mind that don't push expo folder to GitHub.

**Pull request merged**